### ISSUE TITLE
[HOLD] Accessibility: License dropdown

### ISSUE THREAD

**Issue by astridu** [sul-dlss/hungry-hungry-hippo#2089]
I didn't catch this during the last workcycle, but I just noticed the section headers for each of the set of licenses is light gray on white (in both the item and collection forms). Can we change it to bold black? Also, when the Voiceover reads through the list, it says the "[section header name] dimmed". Could we remove the 'dimmed'?

**Comment by astridu** [comment]
I had to do a little video bc I couldn't get a screenshot.

https://github.com/user-attachments/assets/7d5f2a4b-1f51-48f4-9078-4d071afb266a

**Comment by justinlittman** [comment]
I don't think we want to or are obligated to work around the implementation of major browsers. That seems fragile and unnecessarily complex.
